### ISSUE TITLE
CRUD Articles, Authors models

### ISSUE THREAD

**Issue by jannypie** [fifthbymidwest/fifthbymidwest.github.io#14]
Question: Aim to make them API-accessed or just straight roll for now?

As an authorized, authenticated user, I want the ability to create and view articles (posts) so that I can share my ideas.

Articles
- Author
- Date
- Permalink
- Body

Web page view
- See list of articles
- Create an article
- Edit an article
- Delete an article
- See an individual article

**Comment by vaughncm** [comment]
If I wanted to chunk out work on this would your preference be multiple PRs or a long-running PR that can be tracked as I worked on it with updates?

**Comment by jannypie** [comment]
I don't have a preference, do you have one that you like best?

**Comment by vaughncm** [comment]
Nah, I'll start throwing something together tomorrow and if we figure out how best to collaborate on it in the meantime we can do it then.
